ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/defendant/DefendantMapperTest.java

```diff
@@ -205,4 +210,39 @@ public void mapCountyCourtJudgmentToCCDDefendant() {
         assertEquals(ccdCountyCourtJudgment.getRequestedDate(), claimWithCCJ.getCountyCourtJudgmentRequestedAt());
     }
 
+    @Test
+    public void mapToCCDDefendantWithNullSettlement() {
+        //Given
+        TheirDetails theirDetails = SampleTheirDetails.builder().organisationDetails();
+        Claim claimWithCCJ = SampleClaim.getWithSettlement(null);
+
+        //When
+        CCDDefendant ccdDefendant = mapper.to(theirDetails, claimWithCCJ);
+
+        //Then
+        assertNull(ccdDefendant.getSettlementPartyStatements());
+    }
+
+    @Test
+    public void mapToCCDDefendantWithSettlements() {
+        //Given
+        TheirDetails theirDetails = SampleTheirDetails.builder().organisationDetails();
+        Claim claimWithSettlement = SampleClaim
+            .getWithSettlement(
+                SampleSettlement.builder().withPartyStatements(
+                    SamplePartyStatement.offerPartyStatement,
+                    SamplePartyStatement.acceptPartyStatement)
+                    .build());
+        final LocalDateTime settlementReachedAt = claimWithSettlement.getSettlementReachedAt();
+
+        //When
+        CCDDefendant ccdDefendant = mapper.to(theirDetails, claimWithSettlement);
+
+        //Then
+        assertNotNull(ccdDefendant.getSettlementPartyStatements());
+        assertNotNull(ccdDefendant.getSettlementReachedAt());
+        assertThat(ccdDefendant.getSettlementPartyStatements().size(), is(2));
+        assertEquals(settlementReachedAt, ccdDefendant.getSettlementReachedAt());
+    }
+
```

Comment:
Please add unit test to map back from CCD to CMC for the same.

Reply:
done